_posts/2019-06-05-uncertainty-estimates-tfprobability/uncertainty_tfprobability.Rmd

```diff
@@ -0,0 +1,782 @@
+---
+title: "Adding uncertainty estimates to Keras models with tfprobability"
+description: > 
+   As of today, there is no mainstream road to obtain uncertainty estimates from neural networks. All that can be said this that normally, approaches tend to be Bayesian in spirit, involving some way of putting a prior over model weights. This holds true as well for the method presented in this post: We show how to use tfprobability, the R interface to TensorFlow Probability, to add uncertainty estimates to a Keras model in an elegant and conceptually plausible way.
```

Comment:
I think you might cut _This holds true as well for the method presented in this post:_ from the above, since it's implied that you're describing what's contained in the post itself.